pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -184,6 +184,38 @@ public void testQuotedStrings() {
             .getIdentifier().getName(), "Martha''s Vineyard");
   }
 
+  @Test
+  public void testExtract() {
+    {
+      // Case 1 -- Year and date format ('2017-06-15')
+      PinotQuery pinotQuery = CalciteSqlParser.compileToPinotQuery("SELECT EXTRACT(YEAR FROM \"2017-06-15\")");
```

Comment:
Please use single quotes and assert that it is literal (or change it to `timestampColumn` without quotes, which is an identifier). We may have half of the test cases with literal, and the other half with identifier